mpas_analysis/obs/observational_datasets.xml

```diff
@@ -0,0 +1,947 @@
+      analyses. The HADISST1 spanned 1870 onward but the OI.v2, which started
+      in November 1981, better resolved features such as the Gulf Stream and
+      Kuroshio Current which are important components of the climate system.
+      Since the two data sets used different development methods, anomalies
+      from a base period were used to create a more homogeneous record. Also,
+      additional adjustments were made to the SIC data set.
+    </description>
+    <source>
+      [NCAR Hadley-NOAA/OI SST website]
+      (https://climatedataguide.ucar.edu/climate-data/merged-hadley-noaaoi-sea-surface-temperature-sea-ice-concentration-hurrell-et-al-2008)
+    </source>
+    <releasePolicy>
+      Unknown
+    </releasePolicy>
+    <references>
+      [Hurrell et al. (2008)](https://doi.org/10.1175/2008JCLI2292.1)
+    </references>
+    <bibtex>
+      @article{Hurrell2008,
+      author = {James W.  Hurrell  and  James J.  Hack  and  Dennis  Shea  and  Julie M.  Caron  and  James  Rosinski},
+      title = {A New Sea Surface Temperature and Sea Ice Boundary Dataset for the Community Atmosphere Model},
+      journal = {Journal of Climate},
+      volume = {21},
+      number = {19},
+      pages = {5145-5153},
+      year = {2008},
+      doi = {10.1175/2008JCLI2292.1},
+      URL = {https://doi.org/10.1175/2008JCLI2292.1}
+      }
+    </bibtex>
+    <dataUrls>
+      - ftp://ftp.cgd.ucar.edu/archive/SSTICE/MODEL.SST.HAD187001-198110.OI198111-201712.nc
+    </dataUrls>
+    <preprocessing>
+    </preprocessing>
+    <tasks>
+      - climatologyMapSST
+    </tasks>
+  </observation>
+
+  <observation>
+    <name>
+      SSS from NASA Aquarius satellite
+    </name>
+    <component>
+      ocean
+    </component>
+    <description>
+      Level 3 Aquarius sea surface salinity (SSS) data products have a temporal
+      resolutions of daily, 8 day, monthly, 3 months, and annual.  Monthly and
+      seasonal climatology
+      products from Aqaurius are also available. The Aquarius instrument
+      provides global coverage every 7 days. L3 products are gridded at 1
+      degree spatial resolution.
+    </description>
+    <source>
+      [NASA Aquarius FTP server]
+      (ftp://podaac-ftp.jpl.nasa.gov/allData/aquarius/L3/mapped/V4/7day_running/SCI/)
+    </source>
+    <releasePolicy>
+      NASA data are not copyrighted; however, when you publish our data or
+      results derived therefrom, we request that you include an acknowledgment
+      within the text of the publication and reference list.
+      [Data Citation and Acknowledgements]
+      (https://podaac.jpl.nasa.gov/CitingPODAAC)
+    </releasePolicy>
+    <references>
+      [Lagerloef et al. (2015)](ftp://podaac.jpl.nasa.gov/SalinityDensity/aquarius/docs/v4/AQ-014-PS-0016_AquariusSalinityDataValidationAnalysis_DatasetVersion4.0and3.0.pdf)
+    </references>
+    <bibtex>
+    </bibtex>
+    <dataUrls>
+      (missing)
```

Comment:
@milenaveneziani, I need help finding this URL.  I've wandered around the Aquarius FTP server with no luck.  We need to find `Aquarius_V3_SSS_Monthly.nc` (or its V4 or V5 equivalent).

Reply:
Apparently this file/data is no longer available, cannot find it anywhere either. But I read a bit of the SSS data that is available now, and this seems to be interesting:
https://podaac.jpl.nasa.gov/dataset/AQUARIUS_L4_OISSS_IPRC_7DAY_V4?ids=ProcessingLevel&values=*4*&search=Aquarius
It's a blended product where the OI takes into account ARGO data as well, to correct/amend where the satellite data has a high uncertainty/error. It's also higher resolution, 0.5degx0.5deg.
It's only available at a 7-daily product (many 7-day files), so we would have to compute the monthly data ourselves as preprocessing.

Reply:
Okay, so it sounds like we just leave this as "missing" for the time being and we'll do a whole process down the road where we make a preprocessing script that hopefully can download the data (if needed) and process it into a monthly climatology.  Definitely a longer-term project...

Reply:
the older one was all monthly data between 2011 and 2015 in one single file, and MPAS-Analysis would do the climatology, right?

Reply:
> MPAS-Analysis would do the climatology, right?

Yes, that's right.